python/orchestrate/convert.py

```diff
@@ -11,3 +12,20 @@
 ConvertFhirR4ToOmopResponse = bytes
 
 ConvertX12ToFhirR4Response = Bundle
+
+
+def generate_convert_combined_fhir_bundles_request_from_bundles(
+    fhir_bundles: list[Bundle],
+) -> str:
+    """
+    Converts a list of FHIR bundles into a request body for the combine FHIR bundles endpoint.
+
+    ### Parameters
+
+    - `fhir_bundles`: (list[Bundle]): A list of FHIR bundles.
+
+    ### Returns
+
+    The content of the request for the combined FHIR bundles endpoint.
+    """
+    return "\n".join([json.dumps(bundle) for bundle in fhir_bundles])
```

Comment:
two questions here:

1. does `json.dumps` automatically format the output?
2. does `json.dumps` produce valid FHIR json? this might not be a valid question if there's no FHIR abstraction that handles both XML and JSON in python

Reply:
1. format how?
2. This is going from a named map to JSON--there's no python-y way to convert to XML:
    
![image](https://github.com/CareEvolution/OrchestrateSDK/assets/4954394/2f50e9ef-2a78-4906-bbe8-82dc88ce3eb4)

I'm importing from `_internal` here just as an example: the internal type should recognize an equivalent `TypedDict` list passed to the function.

Reply:
1. format as in pretty-print the JSON, which would break the ndjson thing. looks like it does not so all good
2. apparently this is my dotnet showing and doesn't apply to python

Reply:
I dunno why I didn't just do it. Here's a better example.

![image](https://github.com/CareEvolution/OrchestrateSDK/assets/4954394/7483e4e7-6589-4760-a757-39bb7b380f9a)